### PR TITLE
Add automated accessibility tests and fix GSA accessibility scan items

### LINKED ISSUE #3121: 21st Century IDEA Baseline Compliance Criteria - Accessible
**Criteria**

- [x] Routine review of all site content (both pages and files) for conformance to the most current 508 Standards
- [x] Quick remediation of all accessibility issues when they are discovered
- [ ] Incorporation of accessibility testing into the publication process

### LINKED ISSUE #3599: Address accessibility issues (alt text) from AMP scan
## TTS Digital Council provided automated AMP scans and requires fixes by August 31st. 
Our site's issues are all related to alternate text for images: 38 images either have missing alt text, or it's uninformative such as a url for the text.
[AMP scan spreadsheet](https://docs.google.com/spreadsheets/d/19IgCzm3KxAy57TzkCmxRGitIvFhurI4p/edit#gid=1587429197)

## This issue is done when
- [ ] Alt attribute is either empty because the image is purely decorative, or has informative text

### PR DESCRIPTION
**Pull request by neilmb**
# Pull request summary
This adds automated accessibility testing with pa11y to the CircleCI configuration for this project. It also addresses some of the accessibility items that were surfaced on the GSA accessibility scan spreadsheet through some very small template changes.

If successful, this should close #3121 (which is the next-to-last task on #3130) and  close #3599.

It is unlikely that the CircleCI scan will pass the first time it runs on this branch, but it should be possible to patch up every one of the pa11y errors that are identified when that runs. Optimally, the pa11y CircleCI test will pass cleanly on this branch.

## Reminder - please do the following before assigning reviewer

- [ ] update readme
- [ ] For frontend changes, ensure design review

And make sure that automated checks are ok

- fix houndci feedback
- ensure tests pass
- federalist builds
- no new SNYK vulnerabilities are introduced

### CODE REVIEW

**Review by jstrothman — CHANGES_REQUESTED** [review]
@neilmb Awesome to have pa11y running!
One change request: 
For the What We Deliver list of agencies we've partnered with, could you change all the partner logos to be empty alt attributes: alt=" " instead of the accurate pa11y text.
Currently a screenreader user who wants to hear the agencies we've worked with would need to listen to the 48 names twice, e.g.:
"Alaska Department of Health and Social Services logo
Alaska Department of Health and Social Services"

**Review by jstrothman — COMMENTED** [review]
@neilmb Thank you for making the change to the logos alt text. Looks great.
I'm not sure why the build is failing, but other than that, I'm ready to approve/merge

**Comment by neilmb** [comment]
@jstrothman The build is failing because I added pa11y scanning to the test suite. So we will need to make the existing site pass the pa11y scan fully in order to get the tests passing again. But after we get that work done now, passing pa11y will be checked for all new changes. 🎉

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,10 +3,34 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: circleci/ruby:2.7-node
+      - image: cimg/ruby:2.7-node
```
Comment (adunkman) [inline .circleci/config.yml:6]: [non-blocking] Using `cimg/ruby:2.7-browsers` might allow you to skip installing chrome dependencies below. [The `-browsers` variant also includes `node`](https://circleci.com/developer/images/image/cimg/ruby#browsers).

**Review by adunkman — APPROVED** [review]
Looks great! One potential improvement; take it or leave it. 🙂

**Review by jstrothman — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (neilmb) [reply .circleci/config.yml]: Thank you, this is just what I wanted! I don't need to say how long I spent earlier looking for a `-node-browsers` version.

**Pull request merged**